spec/calendar.html

```diff
@@ -459,8 +459,9 @@ <h1>ResolveISOMonth ( _fields_ )</h1>
         It returns the integer month.
       </p>
       <emu-alg>
-        1. Let _month_ be ? Get(_fields_, *"month"*).
-        1. Let _monthCode_ be ? Get(_fields_, *"monthCode"*).
+        1. Note: _fields_ is created in PrepareTemporalFields, so the Get calls below can not fail.
```

Comment:
that's not something that's possible for this AO to know for certain, since anything could call it. It may be true that currently the only callers of ResolveISOMonth meet this contract, but that doesn't mean it's something this AO can assert.

Reply:
I get this concern, but on the other hand it would mean keeping a `?` in a place where it would be untestable. It would be impossible to write a test262 test that could observe an exception being thrown by these steps.

Would it address your concern if it was "Assert: _fields_ was created by PrepareTemporalFields" or "Assert: _fields_ is an ordinary object with no more and no less than the own data properties listed in Table 13" or something like that?

Reply:
@bakkot since this was in response to your review, do you have thoughts on this?

Reply:
I disagree with Jordan on this question; this note seems perfectly fine to me.

@ljharb would you be satisfied if this line were an assertion instead of a note? We have existing assertions along the lines of "x is an ordinary object" which justify the use of `!` in an algorithm; this seems no stranger.

Reply:
I made two suggestions above, would either of those work? Particularly "Assert: _fields_ is an ordinary object with no more and no less than the own data properties listed in [Table 13](https://tc39.es/proposal-temporal/#table-temporal-field-requirements)" seems to me like it explicitly mentions the reason why the Get wouldn't throw.

Reply:
Either of those suggestions seem fine to me, as does simply replacing the Note with an Assert with the current wording.